.github/workflows/test_cupy.yml

```
@@ -0,0 +1,32 @@
+name: test_cupy
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  test_cupy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - uses: Jimver/cuda-toolkit@v0.2.4
+        id: cuda-toolkit
+        with:
+          cuda: '11.2.2'
+      - run:
+          echo "Installed cuda version is ${{steps.cuda-toolkit.outputs.cuda}}"
+      - run:
+          echo "Cuda install location is ${{steps.cuda-toolkit.outputs
+          .CUDA_PATH}}"
+      - run: nvcc --version
+      - run: sudo apt-get update
+      - run: sudo apt-get install --fix-missing ffmpeg python3-soundfile
+      - run: pip install pyre-check pytest torchvision

```

```suggestion
      - run: pip install pyre-check pytest cupy-cuda112

```

You need to actually install cupy :) And you shouldn't need torchvision if you're only running the audio tests.